bsdlib/doc/CHANGELOG.rst

```diff
@@ -5,6 +5,17 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+bsdlib 0.6.1
+************
+
+* Implement hostname verification.
+* Report correct number of avaiable sockets for the application.
```

Comment:
The number of available sockets hasn't changed so I suggest:
`Fixed a bug where the application couldn't allocate the 8th socket.`

Reply:
You fixed the issue by adding one socket so we can report 8 available sockets for the application.  So yes, the number of sockets has changed, but it's internal so will use your text.